packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -295,6 +295,14 @@ - (void)handleCreateMethodCall:(FlutterMethodCall *)call
   });
 }
 
+// Returns integer value if provided and positive, or zero if not specified or negative.
```

Comment:
can you reflect this idea in the function name?

Reply:
renamed to `toPositiveOrNull`